### ISSUE TITLE
Zero everywhere in D_cba

### ISSUE THREAD

**Issue by andthewatersays** [konstantinstadler/pymrio#73]
Hello,

First, thank you for all your packages. It is very useful to understand and practice MRIO.
When I start to handle packages, I have a problem. I have modified the Y matrix in order to correspond with my study case. Once I launch the .calc_all(), I want to see the results with the D_cba matrix, but all the fiels are equal to zeros ... I do not know why because my final demande is not null.
But when I am looking for the D_cba_reg for example, I have results (even if I put zero for the final demande for the region 'AT', I have some results for this region). I do not understand why too.

If you have some suggestions about what I could miss or misunderstood, I will take it !

Thanks !

**Comment by konstantinstadler** [comment]
Hi, 
- for the D_cba vs D_cba_reg difference: You might have household emissions which are added to the regional footprint
- for the other questions you need to give me more information: code, which mrio, ... Ideally, you could make a minimal example with the test system inlcuded in pymrio (test_io = pymrio.load_test()) which makes debuging much faster
- I will be away until August, so please expect some delays in my anwers

**Comment by andthewatersays** [comment]
Thanks for your answer.

Ok, I will try to work with the test in order to understand why there is a problem before asking other technical questions.
But I have one question (which could look basic, but I would like to be sure that I modify right things) : the matrix that I have to modify in order to enter my data, it is the Y matrix, is it not ?

**Comment by konstantinstadler** [comment]
Yes. Either in place or you assign it back with
IO.Y=Y

**Comment by andthewatersays** [comment]
I think that I found what is going on. Indeed, before launched the first calculation with _.calc_all()_, I initialized the Y matrix with zero everywhere. I launched _.calc_all()_ and I had the D_cba matrix null (and it is normal). But just after I modified the Y matrix and I launched again the _.calc_all()_ (this is my error I think) and the D_cba matrix was still null.
Because I tested modifying the Y matrix before launching _.calc_all()_ and in this case, it works directly. So I think that the fact initializing the Y matrix was the problem, as if there is just only version possible of the Y matrix by simulation.

I hope I will be clear with my poor English.
